src/layout/Layout.jsx

```diff
@@ -5,18 +5,23 @@ import "react-toastify/dist/ReactToastify.css";
 
 import Footer from "@/components/Footer";
 import Navbar from "@/components/Navbar/Navbar";
+import { motion } from "framer-motion";
 
 export default function Layout({ children }) {
-    // Put Header or Footer around the children element
-    // Example
     return (
         <>
             <Navbar />
+            <motion.div
+                initial={{ opacity: 0, y: 15 }}
+                animate={{ opacity: 1, y: 0 }}
+                exit={{ opacity: 0, y: 15 }}
+                transition={{ delay: 0.25 }}
+            >
+                {children}
+            </motion.div>
             <ToastContainer position='top-center' />
             {children}
```

Comment:
I was wondering why it shows the children too time on each other  page .. we need to remove this extra `{children}`

Reply:
it's because the develop was merged to this branch I'm working on fixing it